modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1150,6 +1150,45 @@ definition {
 		AppBuilderAdmin.validateItemNameOnTable(itemName = "Content of form deployed as widget.");
 	}
 
+	@description = "Search for an object in the object list"
+	@priority = "5"
+	test SearchForObject {
+		property portal.acceptance = "true";
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object 1",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		AppBuilderAdmin.gotoBack();
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object 2",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		AppBuilderAdmin.gotoBack();
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object 3",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		AppBuilderAdmin.gotoBack();
+
+		AppBuilderAdmin.searchByItem(itemName = "object");
+
+		AppBuilderAdmin.chooseSearchFilterByOption(filterOption = "Name");
```

Comment:
Hey @nkkwon could you please validate if the objects are being changing the order when switching between filter option by Name and Modified Date ? You can use the macro AppBuilderAdmin.validateListTitleAtRow().

Reply:
Hey @nkkwon , I saw that you've added a validation after filter by Modified Date, but It's missing the validation after change the filter by Name.

Reply:
Pushed new changes for the validation